docs/pages/faq.mdx

```diff
@@ -63,6 +63,31 @@ are not logged in to Teleport or may not have access to a browser.
 Make sure that all Teleport components are at least at version 10.3.0. Older versions
 don't support the SFTP protocol, and it's enabled by default in `tsh` v11.0.0 and OpenSSH v9.0.
 
+## How do I extract certificates from a container running a distroless image?
+
+Extracting certificates created with `tctl auth sign` from a container running
+a distroless image can be tricky due to the absence of a shell and other OS tools. 
+
+Where possible, you should log into the cluster using `tsh` and use `tctl auth sign`
+locally to generate certificates. This way the action will be logged against your
+Teleport user and be subject to all of the usual Teleport RBAC policies in your
+cluster.
+
+If this is not possible, use `tctl auth sign --tar` to collect all the files
+generated by `tctl auth sign` into a `tar` archive, which is streamed directly
+to `stdout`. The resulting certificates are never stored on the container
+filesystem. You can either pipe this output directly to `tar`, or redirect it
+to a local file for later use.
+
+For example:
+
+```code
+kubectl exec ${KUBECTL_FLAGS} -- tctl auth sign --user alice --format tls -o alice.local --tar | tar xv
+x alice.local.crt
+x alice.local.key
+x alice.local.cas
+```
+
```

Comment:
Just wondering if there should be a pointer to the `tbot` docs here also? Thoughts?

Reply:
What did you have in mind for the link to the `tbot` docs?